packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -717,7 +732,16 @@ class ScrollableState extends State<Scrollable> with TickerProviderStateMixin, R
       );
     }
 
-    return _configuration.buildViewportChrome(context, result, widget.axisDirection);
+    return _configuration.useDecoration
+      ? _configuration.buildViewportDecoration(
+          context,
+          result,
+          ScrollableDetails(
+            widget.axisDirection,
+            widget.autoScrollbar ? widget.controller : null,
```

Comment:
This is kinda a surprising side effect: Setting a controller in the details gives you scrollbars? We should at least document this. And I am wondering if we should make this more explicit.

Reply:
I'll add more documentation around this.